strings/slices/slices_test.go

```diff
@@ -0,0 +1,281 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package slices
+
+import (
+	"fmt"
+	"reflect"
+	"regexp"
+	"testing"
+
+	"k8s.io/utils/strings/validator"
+)
+
+func Example() {
+	a := []string{
+		"10.0.0.0", "FOO", "1000::1", "fd80::4%eth0", "BAR", "192.168.1.300",
+		"172.12.0.1", "fc00::5000",
+	}
+	v := validator.IP(0)
```

Comment:
This package is so "leaf" that I don't think I would rely on validator in the tests.  I would just open-code some trivial logic here.

Reply:
You are absolutely right. I would not use them myself if I had not written them. Validators removed.